### ISSUE TITLE
Bosh: Session Creation Response add polling attribute twice

### ISSUE THREAD

**Issue by mercuriev** [processone/ejabberd#2790]
On my request for session:

```xml
<body
 xmlns="http://jabber.org/protocol/httpbind"
 to="localhost"
 wait="0"
 hold="0"
 ack="1"
 rid="6510799"/>
```

ejabberd replies with:

```xml
<body
 authid='5439289209245408066'
 xmlns='http://jabber.org/protocol/httpbind'
 sid='efc0d228bd6f1c179c3fb71e786a05ed9651676d'
 wait='0'
 ver='1.11'
 polling='2'
 inactivity='30'
 hold='0'
 xmpp:restartlogic='true'
 requests='1'
 secure='true'
 maxpause='120'
 xmlns:xmpp='urn:xmpp:xbosh'
 xmlns:stream='http://etherx.jabber.org/streams'
 from='localhost'
 polling='2'/>
```

The reply contains `polling` attribute twice which invalidates XML and cause client parser to fail.

**Comment by badlop** [comment]
I've tried with ejabberd 18.12 and also 18.01, with Tkabber client. In both cases, only the first polling attribute is returned; after the "from" attribute, the second polling is not returned. What ejabberd version are you using, how did you install it, and what client?

**Comment by mercuriev** [comment]
I'm using latest docker image `ejabberd/ecs` with `18.12.1` inside.
Connecting with PHP code writing to open socket.

I noticed that it only happens when `wait` is `0` in request.

**Comment by badlop** [comment]
Ah, right, looking at ejabberd_bosh.erl:
- in line 327, if wait == 0, a _polling_ attribute is added
- and also, in line 335 a _polling_ attribute is added always

https://github.com/processone/ejabberd/blob/master/src/ejabberd_bosh.erl#L327

**Comment by prefiks** [comment]
Fixed in 122cb4b9594041487f9bb18c683531d7105ce2f9
